packages/card/src/spectrum-config.js

```diff
@@ -26,12 +26,12 @@ const config = {
                     name: 'header',
                 },
                 {
-                    selector: '.spectrum-Card-title',
-                    name: 'title',
+                    selector: '.spectrum-Card-heading',
+                    name: 'heading',
                 },
                 {
-                    selector: '.spectrum-Card-subtitle',
-                    name: 'subtitle',
+                    selector: '.spectrum-Card-subheading',
+                    name: 'subheading',
```

Comment:
Ahh -- definitely an issue here. @Westbrook should I revert all the changes here, or just the selector changes?

Reply:
As long as the `name` value here matches the actual class we're using in the template, I'm not sure it matters much what we actually use here as it's part of the fully private CSS API.